src/vivarium_nih_us_cvd/components/causes/causes.yaml

```diff
@@ -0,0 +1,146 @@
+#causes:
+#    ischemic_stroke:
+#        states:
+#            susceptible:
+#                cause_type: "cause"                             # default
+#                transient: False                                # default
+#                allow_self_transitions: True                    # default
+#                side_effect: None                               # default
+#                cleanup_function: None                          # default
+#            acute_ischemic_stroke:
+#                cause_type: "sequela"
+#                transient: False                                # default
+#                allow_self_transitions: True                    # default
+#                side_effect: None                               # default
+#                cleanup_function: None                          # default
+#                get_data_functions:
+#                    prevalence: None                            # default
+#                    birth_prevalence: None                      # default
+#                    # todo figure out how to record this - it's a lambda function in the code
+#                    dwell_time: "28 days"
+#                    disability_weight: None                     # default
+#                    excess_mortality_rate: None                 # default
+#            chronic_ischemic_stroke:
+#                cause_type: "sequela"
+#                transient: False                                # default
+#                side_effect: None                               # default
+#                allow_self_transitions: True                    # default
+#                cleanup_function: None                          # default
+#        transitions:
+#            acute_ischemic_stroke_incidence:
+#                source: "susceptible"
+#                sink: "acute_ischemic_stroke"
+#                data_type: "rate"
+#                get_data_functions:
+#                    incidence_rate: None                        # default
+#                    transition_rate: None                       # default
+#                    remission_rate: None                        # default
+#                    proportion: None                            # default

```

Comment:
Is it still useful to have this commented block?

Reply:
I think it's useful until we have some documentation of what defaults are.